packages/cw-tokenfactory-types/src/cosmwasm.rs

```diff
@@ -0,0 +1,281 @@
+use std::convert::TryFrom;
+use std::convert::TryInto;
+
+use osmosis_std_derive::CosmwasmExt;
+
+// see https://github.com/notional-labs/wasmd/blob/v0.30.0-sdk469.4/proto/cosmwasm/tokenfactory/v1beta1/tx.proto
```

Comment:
Why are we using the types in a fork that probably won't be maintained?

Reply:
Who is using these?

Reply:
migaloo uses `/cosmwasm.tokenfactory...` types. it's not that specific fork necessarily, that's just a link to a tag that has the protobufs. the msg types are the exact same except some of the osmosis msgs have additional fields, but mostly only the type URL differs. migaloo will probably eventually migrate over to the osmosis types, but for now they are on the cosmwasm version 🤷 i don't think it's a big deal to support both, and since they are behind feature flags, all this does is make it compilable for a wider set of chains

Reply:
Oh, sounds good then. Maybe add a comment about which chain it's for? :shipit: